lib/matplotlib/category.py

```diff
@@ -189,11 +210,22 @@ def update(self, data):
         """
         data = np.atleast_1d(np.array(data, dtype=object))
 
+        # check if convertable to number:
```

Comment:
arriving after the battle, but this should have been named "convertible", not "convertable"...